server/utils.py

```diff
@@ -190,13 +197,20 @@ def get_files_by_status(auth_token, status):
     return dict(db_files)
 
 
+def get_user_by_auth_token(auth_token):
+    if g.db.execute('select exists(select 1 from access where auth_token=? LIMIT 1)', (auth_token,)).fetchone()[0]:
+        return g.db.execute('select user_id from access where auth_token=?', (auth_token,)).fetchone()[0]
+    return False
```

Comment:
`False` is a conceptually weird return value here. I'd suggest not explicitly returning anything (implicitly returning `None`) which is still false-like.